frontend/src/pages/users/Users.tsx

```diff
@@ -62,6 +90,7 @@ const UsersComponent: React.FC = () => {
         setPage={setPage}
```

Comment:
This is missing to make the pagination work correctly:
```suggestion
        setPage={setPage}
        perPage={PER_PAGE}
```